src/Hintman/Config.hs

```diff
@@ -0,0 +1,43 @@
+module Hintman.Config
+    ( HintmanConfig(..)
+    , defaultHintmanConfig
+    , loadFileConfig
+    ) where
+
+import Toml (AnyValue (..), BiMap (..), BiToml, (.=))
+
+import qualified Toml
+
+
+
+data Check = TrailingSpaces
+           | TrailingNewlines deriving (Eq, Read, Show)
+
+newtype HintmanConfig = HintmanConfig
+    { hintmanConfigChecks :: [Check]
+    } deriving (Show)
+
+defaultHintmanConfig :: HintmanConfig
+defaultHintmanConfig = HintmanConfig [TrailingSpaces, TrailingNewlines]
+
+showCheck :: Check -> Text
+showCheck TrailingSpaces   = "trailing-spaces"
+showCheck TrailingNewlines = "trailing-newlines"
+
+parseCheck :: Text -> Maybe Check
+parseCheck "trailing-spaces"   = Just TrailingSpaces
+parseCheck "trailing-newlines" = Just TrailingNewlines
+parseCheck _                   = Nothing
+
+hintmanConfiguationT :: BiToml HintmanConfig
+hintmanConfiguationT = HintmanConfig
+    <$> Toml.arrayOf _Check "checks" .= hintmanConfigChecks
+  where
+    _Check :: BiMap AnyValue Check
+    _Check = BiMap
+         { forward = \(AnyValue t) -> Toml.matchText t >>= parseCheck
+         , backward = Just . AnyValue . Toml.Text . showCheck
+         }
+
+loadFileConfig :: MonadIO m => FilePath -> m HintmanConfig
+loadFileConfig = Toml.decodeFile hintmanConfiguationT
```

Comment:
I feel like somewhere at this point you should return `defaultHintmanConfig` if file is not present or if there `LoadTomlException`. Just print debugging info to the terminal and return `defaultHintmanConfig`.